Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -1232,11 +1232,11 @@ CSSScrollbarWidthEnabled:
   humanReadableDescription: "Enable scrollbar-width CSS property"
   defaultValue:
     WebKitLegacy:
-      default: false
```

Comment:
We should not have feature in `testable` status activated by default

Reply:
Right fixed.